verifact/forum/schema.py

```diff
@@ -10,6 +10,12 @@ class Meta:
         interfaces = (relay.Node,)
 
 
+class AnswerNode(DjangoObjectType):
```

Comment:
Looks like somewhere you lost the `AnswerConnection`, throws errors if you try to use the API